ext/mysqlnd/mysqlnd_enum_n_def.h

```diff
@@ -234,6 +234,7 @@ typedef enum mysqlnd_client_option
 	MYSQL_REPORT_DATA_TRUNCATION,
 	MYSQL_OPT_RECONNECT,
 	MYSQL_OPT_SSL_VERIFY_SERVER_CERT,
+	MYSQL_OPT_TLS_VERSION,
```

Comment:
Hi, I just pushed a change to this file with this and other enums added (coming from MySQL 5.7)